Makefile

```diff
@@ -123,6 +127,31 @@ generate-stackgres-crds:
 	go run sigs.k8s.io/controller-tools/cmd/controller-gen object paths=./apis/stackgres/v1beta1/...
 	rm apis/stackgres/v1/*_crd.yaml
 
+.PHONY: generate-cnpg-crds
+generate-cnpg-crds:
+	# curl -L ${CNPG_CRD_URL} > apis/cnpg/v1/all_crd.yaml
```

Comment:
is this needed or not?

Reply:
It's kept for reference, if we ever need to re-generate things for $reasons.